Gateway/Request/PaymentOffsiteBuilder.php

```diff
@@ -49,10 +49,10 @@ public function build(array $buildSubject)
         
         switch ($method->getMethod()) {
             case Alipay::CODE:
-                $paymentInfo[self::OFFSITE] = 'alipay';
+                $paymentInfo[self::SOURCE_TYPE] = 'alipay';
```

Comment:
@jacekstan this one a bit strange that we have `source[type]` as a string while in fact, `type` should be assigned in an array.. how about we assign this one as array instead? 
Something like
```php
$paymentInfo[self::SOURCE] = [
    'type' => 'alipay'
]
```

or 
```php
$paymentInfo[self::SOURCE] = [
    self::SOURCE_TYPE => 'alipay'
]
```